Source/WTF/wtf/text/StringCommon.h

```diff
@@ -260,8 +270,8 @@ ALWAYS_INLINE bool equal(const Latin1Character* a, std::span<const Latin1Charact
     length &= 3;
 
     if (length) {
-        const Latin1Character* aRemainder = byteCast<Latin1Character>(a);
-        const Latin1Character* bRemainder = byteCast<Latin1Character>(b);
+        const CharacterType* aRemainder = byteCast<CharacterType>(a);
+        const CharacterType* bRemainder = byteCast<CharacterType>(b);
```

Comment:
If we did need to modify this, and I’m not sure we do, I suggest using auto:
```suggestion
        auto* aRemainder = byteCast<CharacterType>(a);
        auto* bRemainder = byteCast<CharacterType>(b);
```